pkg/utils/runtimes.go

```diff
@@ -52,7 +52,7 @@ func AddRuntimesIfNotExist(runtimes []data.Runtime, newRuntime data.Runtime) (up
 	return updatedRuntimes
 }
 
-// Get Alluxio Runtime object given name and namespace
+// GetAlluxioRuntime get Alluxio Runtime object given name and namespace
```

Comment:
-> GetAlluxioRuntime `gets` Alluxio Runtime object `with` the  given name and namespace